cpp/ql/test/library-tests/rangeanalysis/SimpleRangeAnalysis/test.c

```diff
@@ -638,4 +638,16 @@ void two_bounds_from_one_test(short ss, unsigned short us) {
   if (ss + 1 < sizeof(int)) {
     out(ss); // -1 .. 2
   }
-}
\ No newline at end of file
+}
+
+void guard_bound_out_of_range(void) {
+  int i = 0;
+  if (i < 0) {
+    out(i); // 0 .. 0 [BUG: is -max .. +max]
```

Comment:
This is unreachable, should there be any bounds?

Reply:
You're right. Fixed.